src/app/controller/new.ts

```diff
@@ -28,7 +28,7 @@ class CreateAppController extends Controller {
       csrfToken: await ctx.getCsrf(),
       msg: ctx.query.msg,
       error: ctx.query.error,
-      name: ctx.query.name,
+      name: ctx.query.nickname,
```

Comment:
Would you mind taking this 1 step further and also rename the property from `name` to `nickname` so it's consistent throughout the application? This likely also means updating the template